contracts/contracts/subnet/SubnetActorCheckpointingFacet.sol

```diff
@@ -127,9 +114,17 @@ contract SubnetActorCheckpointingFacet is SubnetActorModifiers, ReentrancyGuard,
     /// @param checkpointHeight The height of the checkpoint containing the committed messages.
     /// @param inclusions An array of inclusion proofs and messages to be executed.
     function execBottomUpMsgBatch(
-        uint256 checkpointHeight,
+        uint64 checkpointHeight,
+        StateCommitmentBreakDown calldata breakdown,
         BottomUpBatch.Inclusion[] calldata inclusions
     ) external whenNotPaused {
+        validateAppHash(checkpointHeight, breakdown);
+
+        LibBottomUpBatch.recordBottomUpBatchCommitment(checkpointHeight, breakdown.msgBatchCommitment);
+        _execBottomUpMsgBatch(checkpointHeight, inclusions);
```

Comment:
This won’t allow multi-phase execution, i.e., execution without the full set of inclusion proofs.

Reply:
Is multi-phase execution possible before this change?